typha/metadata.csv

```diff
@@ -0,0 +1,31 @@
+typha.breadcrumb.seq.number,gauge,,,,Current (server-local) sequence number; number of snapshot deltas processed.,0,typha,typha.breadcrumb.seq.number,,
+typha.breadcrumb.size.count,count,,,,Number of KVs recorded in each breadcrumb.,0,typha,,,
+typha.breadcrumb.size.sum,count,,,,Number of KVs recorded in each breadcrumb.,0,typha,,,
+typha.client.latency.secs.count,count,,,,Per-client latency. I.e. how far behind the current state is each client.,0,typha,,,
+typha.client.latency.secs.sum,count,,,,Per-client latency. I.e. how far behind the current state is each client.,0,typha,,,
+typha.client.snapshot.send.secs.count,count,,,,How long it took to send the initial snapshot to each client.,0,typha,,,
+typha.client.snapshot.send.secs.sum,count,,,,How long it took to send the initial snapshot to each client.,0,typha,,,
+typha.client.write.latency.secs.count,count,,,,Per-client write. How long each write call is taking.,0,typha,,,
+typha.client.write.latency.secs.sum,count,,,,Per-client write. How long each write call is taking.,0,typha,,,
+typha.connections.accepted,count,,,,Total number of connections accepted over time.,0,typha,typha.connections.accepted,,
+typha.connections.active,gauge,,,,Number of open client connections (including connections that have not completed the handshake).,0,typha,typha.connections.active,,
+typha.connections.streaming,gauge,,,,Number of client connections that are actively streaming (i.e. connections that successfully completed the handshake).,0,typha,typha.connections.streaming,,
+typha.connections.dropped,count,,,,Total number of connections dropped due to rebalancing.,0,typha,typha.connections.dropped,,
+typha.kvs.per.msg.count,count,,,,Number of KV pairs sent in each message.,0,typha,,,
+typha.kvs.per.msg.sum,count,,,,Number of KV pairs sent in each message.,0,typha,,,
+typha.log.errors,count,,,,Number of errors encountered while logging.,0,typha,typha.log.errors,,
+typha.logs.dropped,count,,,,Number of logs dropped because the output stream was blocked.,0,typha,typha.logs.dropped,,
+typha.next.breadcrumb.latency.secs.count,count,,,,Time to retrieve next breadcrumb when already behind.,0,typha,,,
+typha.next.breadcrumb.latency.secs.sum,count,,,,Time to retrieve next breadcrumb when already behind.,0,typha,,,
+typha.ping.latency.count,count,,,,Round-trip ping/pong latency to client. Typha's protocol includes a regular ping/pong keepalive to verify that the connection is still up.,0,typha,,,
+typha.ping.latency.sum,count,,,,Round-trip ping/pong latency to client. Typha's protocol includes a regular ping/pong keepalive to verify that the connection is still up.,0,typha,,,
+typha.updates.skipped,count,,,,Total number of updates skipped because the datastore change was not relevant. (For example an update to a Kubernetes Pod field that Calico does not read.),0,typha,typha.updates.skipped,,
```

Comment:
```suggestion
typha.updates.skipped,count,,,,Total number of updates skipped because the datastore change was not relevant. (For example, an update to a Kubernetes Pod field that Calico does not read.),0,typha,typha.updates.skipped,,
```